packages/components/src/ObjectViewer/JSONLike/JSONLike.test.js

```diff
@@ -184,6 +208,22 @@ describe('JSONLike', () => {
 			// expect
 			expect(wrapper.getElement()).toMatchSnapshot();
 		});
+		it('should render without tooltip', () => {
+			// when
+			const wrapper = shallow(
```

Comment:
please  use react testing library. enzyme is deprecated and block us to update to react 18

Reply:
Thanks, I updated the test cases with RTL.

Reply:
Thanks !